src/main/java/jp/co/soramitsu/sora/didresolver/DTO/Service.java

```diff
@@ -0,0 +1,27 @@
+package jp.co.soramitsu.sora.didresolver.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+
+/**
+ * @author sbt-rogachev-sn
```

Comment:
is this comment necessary? 
IMO it is better to remove autogenerated comments like this

Reply:
remove comments